pages/events/index.vue

```diff
@@ -234,6 +241,10 @@ export default class extends QiskitPage {
 
   &__no-events-msg {
     @include type-style('body-short-02');
+
+    a {
```

Comment:
Just to have in mind: We should change this element selector with a class selector when we will do the BEM refactor :)